### PR TITLE
Update german translation

### PR DESCRIPTION
**Pull request by SoftCreatR**
Add missing language variables.

### CODE REVIEW

**Comment by SoftCreatR** [comment]
@TorstenDittmann Is this supposed to fail? :D

**Comment by PineappleIOnic** [comment]
Thank you so much for the PR 🤩. We're adding the `hacktoberfest-accepted` label to ensure this PR counts towards your Hacktoberfest contributions count. With that said, please stay active on this PR to address any comments once you receive a review. Happy Hacktoberfest! 🎃

**Comment by johannesschobel** [comment]
LGTM

**Comment by stnguyen90** [comment]
@TorstenDittmann would you be able to review and merge this?

File: app/config/locale/translations/de.json [hunk 1]
```diff
@@ -27,6 +27,12 @@
     "emails.invitation.footer": "Wenn du nicht interessiert bist, kannst du diese Nachricht ignorieren.",
     "emails.invitation.thanks": "Danke",
     "emails.invitation.signature": "{{project}}-Team",
+    "emails.certificate.subject": "Zertifikatsfehler für %s",
+    "emails.certificate.hello": "Hallo",
+    "emails.certificate.body": "Das Zertifikat für Ihre Domain '{{domain}}' konnte nicht generiert werden. Dies ist der {{attempt}}. Versuch. Fehlermeldung: {{error}}",
```
Comment (TorstenDittmann) [inline app/config/locale/translations/de.json:32]:
```suggestion
    "emails.certificate.body": "Das Zertifikat für ihre Domain '{{domain}}' konnte nicht generiert werden. Dies ist der {{attempt}}. Versuch. Fehlermeldung: {{error}}",
```

File: app/config/locale/translations/de.json [hunk 2]
```diff
@@ -27,6 +27,12 @@
...
+    "emails.certificate.footer": "Ihr bisheriges Zertifikat ist noch 30 Tage nach Auftreten des ersten Fehlers gültig. Wir empfehlen dringend, diesen Vorfall zu untersuchen, da eine sichere Kommunikation mit Ihrer Domain nach Ablauf des Zertifikats nicht mehr möglich sein wird.",
```
Comment (TorstenDittmann) [inline app/config/locale/translations/de.json:33]:
```suggestion
    "emails.certificate.footer": "Ihr bisheriges Zertifikat ist noch 30 Tage nach Auftreten des ersten Fehlers gültig. Wir empfehlen dringend, diesen Vorfall zu untersuchen, da eine sichere Kommunikation mit ihrer Domain nach Ablauf des Zertifikats nicht mehr möglich sein wird.",
```

**Review by TorstenDittmann — CHANGES_REQUESTED** [review]

**Comment by SoftCreatR** [comment]
@TorstenDittmann Wie kommst du denn darauf, dass bei der formellen Anrede `Ihre` klein geschrieben wird? Oder ist die Anrede hier grundsätzlich informell? Denn dann muss ich das Wording sowieso ändern.

**Comment by SoftCreatR** [comment]
Vergiss die Frage wieder. Ich habe die Übersetzung damals ja selbst gemacht und die Anrede informell gehalten. Ich habe die Texte entsprechend überarbeitet.

**Comment by christyjacob4** [comment]
@SoftCreatR thanks a lot for your contributions during Hacktoberfest 2022! 

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.

**Pull request closed without merging**